models/fizzbuzzer.rb

```diff
@@ -0,0 +1,44 @@
+class Fizzbuzzer
+
+  attr_accessor :range
+  attr_writer :fizzer, :buzzer
+
+  def initialize(count)
+    @range = (1..count)
+  end
+
+  def run
+    @range.each do |number|
+      fizz = fizzes?(number)
+      buzz = buzzes?(number)
+      case
+        when fizz && buzz
+          puts 'Fizzbuzz!'
```

Comment:
dont do the `puts` here.. return the value.. It makes it easier to test
do the puts in the iterator